packages/lou_bank_ui/lib/src/widgets/lou_button.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+
+import 'package:lou_bank_ui/lou_bank_ui.dart';
+
+enum LouButtonVariant {
+  darkGrey,
+  yellow,
+}
+
+class LouButton extends StatelessWidget {
+  const LouButton({
+    Key? key,
+    required this.text,
+    required this.variant,
+    required this.onPressed,
+  }) : super(key: key);
+
+  final String text;
+  final LouButtonVariant variant;
+  final VoidCallback onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return variant == LouButtonVariant.yellow
+        ? ElevatedButton(
+            style: ElevatedButton.styleFrom(
+              primary: LouColors.transparent,
+              onPrimary: LouColors.black,
+              shape: const RoundedRectangleBorder(
+                borderRadius: BorderRadius.all(
+                  Radius.circular(38),
+                ),
+              ),
+              padding: EdgeInsets.zero,
+            ),
+            onPressed: onPressed,
+            child: Ink(
+              decoration: BoxDecoration(
+                gradient: const LinearGradient(
+                  colors: [
+                    LouColors.gradientYellowStart,
+                    LouColors.gradientYellowEnd,
+                  ],
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                ),
+                borderRadius: BorderRadius.circular(38),
+              ),
+              child: Container(
```

Comment:
We don't use containers cuz they have a lot of shit 

Padding -> Align -> SizedBox would be much better. 

If you read container file you will see that it is a widget that has multiple children. Use it only when you use most of them.

Research: ColoredBox, SizedBox, Align, Padding, Center (Align: Aligment.Center) and try to use the simplest widgets